bin2tif/Dockerfile

```diff
@@ -26,4 +20,7 @@ CMD ["extractor"]
 ENV RABBITMQ_EXCHANGE="terra" \
     RABBITMQ_VHOST="%2F" \
     RABBITMQ_QUEUE="terra.stereo-rgb.bin2tif" \
-    MAIN_SCRIPT="terra_bin2tif.py"
+    MOUNTED_PATHS='{"home/clowder/sites":"/home/extractor/sites"}' \
```

Comment:
don't need MOUNTED_PATHS here because we have it in the admin repo YAML file.

Reply:
removed